src/evaluators/state/PHAL_SaveStateField_Def.hpp

```diff
@@ -251,6 +255,9 @@ saveNodeState(typename Traits::EvalData workset)
     default:  // error!
       TEUCHOS_TEST_FOR_EXCEPTION (true, std::runtime_error, "Error! Unexpected field dimension (only node_scalar/node_vector for now).\n");
   }
+#else
```

Comment:
Maybe a run-time error should be thrown here?

Reply:
I guess you're right.